Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -11,12 +11,34 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return message;
+            var output = ""; 
+
+            if (String.IsNullOrEmpty(message))
+            {
+                return message; 
+            }
+
+
+            foreach (char ch in message)
+            {
+                output += DecryptOneChar(ch, shift); 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use a StringBuilder instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5012532)

Reply:
solved